<ui version="4.0" >
 <class>spareImportDialog</class>
 <widget class="QDialog" name="spareImportDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>612</width>
    <height>567</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle" >
   <string>请导入你的系列图像</string>
  </property>
  <property name="sizeGripEnabled" >
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4" >
   <item>
    <widget class="QLabel" name="label" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text" >
      <string>&lt;!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0//EN" "http://www.w3.org/TR/REC-html40/strict.dtd">
&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'宋体'; font-size:9pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-weight:600; color:#ff0000;">1.请在左边的树型框中选择系列图像所在的位置&lt;/span>&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-weight:600; color:#ff0000;">2.请在右边的文件列表框中对文件名进行重命名，排序[当然重命名和排序并不是必须的]&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-weight:600; color:#ff0000;">3.在文件列表框中选择文件，在右上角可以查看预览，在对话框下方展开可以查看相关的头文件信息&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-weight:600; color:#ff0000;">4.确认无误后，点击导入。&lt;/p>&lt;/body>&lt;/html></string>
     </property>
     <property name="buddy" >
      <cstring>openButton</cstring>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3" >
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2" >
       <item>
        <widget class="QPushButton" name="openButton" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize" >
          <size>
           <width>77</width>
           <height>300</height>
          </size>
         </property>
         <property name="text" >
          <string>打开(&amp;O)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="filelistWidget" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout" >
         <item>
          <widget class="QPushButton" name="sortButton" >
           <property name="enabled" >
            <bool>false</bool>
           </property>
           <property name="text" >
            <string>排序(&amp;S)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="upButton" >
           <property name="enabled" >
            <bool>false</bool>
           </property>
           <property name="text" >
            <string>上移(&amp;U)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="downButton" >
           <property name="enabled" >
            <bool>false</bool>
           </property>
           <property name="text" >
            <string>下移(&amp;D)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="deleteButton" >
           <property name="enabled" >
            <bool>false</bool>
           </property>
           <property name="text" >
            <string>删除(&amp;Del)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="renameButton" >
           <property name="enabled" >
            <bool>false</bool>
           </property>
           <property name="text" >
            <string>重命名(&amp;R)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="backButton" >
           <property name="enabled" >
            <bool>false</bool>
           </property>
           <property name="text" >
            <string>重置(&amp;B)</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout" >
       <item>
        <widget class="QToolButton" name="tabButton" >
         <property name="enabled" >
          <bool>false</bool>
         </property>
         <property name="text" >
          <string>图像信息>>></string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer" >
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>317</width>
           <height>17</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QComboBox" name="fileformatcomboBox" >
         <property name="enabled" >
          <bool>false</bool>
         </property>
         <item>
          <property name="text" >
           <string>DICOM</string>
          </property>
         </item>
         <item>
          <property name="text" >
           <string>BMP</string>
          </property>
         </item>
         <item>
          <property name="text" >
           <string>JPG</string>
          </property>
         </item>
         <item>
          <property name="text" >
           <string>Tiff</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="importButton" >
         <property name="enabled" >
          <bool>false</bool>
         </property>
         <property name="text" >
          <string>导入系列图像(&amp;I)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_2" >
         <property name="text" >
          <string>退出(&amp;E)</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QFrame" name="frame" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize" >
        <size>
         <width>256</width>
         <height>256</height>
        </size>
       </property>
       <property name="frameShape" >
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow" >
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2" >
        <item>
         <widget class="QTabWidget" name="tabWidget" >
          <property name="enabled" >
           <bool>false</bool>
          </property>
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize" >
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="currentIndex" >
           <number>0</number>
          </property>
          <widget class="QWidget" name="vtkTab" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <attribute name="title" >
            <string>图像</string>
           </attribute>
           <widget class="QVTKRenderWindowInteractor" native="1" name="previewWidget" >
            <property name="geometry" >
             <rect>
              <x>9</x>
              <y>9</y>
              <width>551</width>
              <height>281</height>
             </rect>
            </property>
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </widget>
          <widget class="QWidget" name="headertab" >
           <attribute name="title" >
            <string>文件头信息</string>
           </attribute>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVTKRenderWindowInteractor</class>
   <extends>QWidget</extends>
   <header>QVTKRenderWindowInteractor</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>filelistWidget</tabstop>
  <tabstop>upButton</tabstop>
  <tabstop>downButton</tabstop>
  <tabstop>renameButton</tabstop>
  <tabstop>backButton</tabstop>
  <tabstop>tabButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>spareImportDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>553</x>
     <y>289</y>
    </hint>
    <hint type="destinationlabel" >
     <x>610</x>
     <y>286</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>importButton</sender>
   <signal>clicked()</signal>
   <receiver>spareImportDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>478</x>
     <y>287</y>
    </hint>
    <hint type="destinationlabel" >
     <x>611</x>
     <y>328</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
